django/forms/formsets.py

```diff
@@ -437,8 +437,4 @@ def formset_factory(form, formset=BaseFormSet, extra=1, can_order=False,
 
 def all_valid(formsets):
     """Return True if every formset in formsets is valid."""
-    valid = True
-    for formset in formsets:
-        if not formset.is_valid():
-            valid = False
-    return valid
+    return all(formset.is_valid() for formset in formsets)
```

Comment:
This change isn't correct as it results in a a behavior change due to short-circuiting. https://github.com/django/django/pull/9579 adds a regression test.